lib/encryption/utils/bootstrap.dart

```diff
@@ -494,12 +494,12 @@ class Bootstrap {
         }
       }
       if (newSsssKey != null) {
-        final storeFutures = <Future<void>>[];
+        Logs().v('Store new SSSS key entries...');
+        // NOTE(TheOneWithTheBraid): do not use Future.wait due to rate limits
+        // and token refresh trouble
         for (final entry in secretsToStore.entries) {
-          storeFutures.add(newSsssKey!.store(entry.key, entry.value));
+          await newSsssKey!.store(entry.key, entry.value);
```

Comment:
Is this related?

Reply:
Yes, the token rotation might otherwise happen during the requests as mentioned in the comment in the line above.

Reply:
I don't get it. Can you please explain?